source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -0,0 +1,205 @@
+
+## [How to migrate to Octane 🐹🏗️](https://guides.emberjs.com/release/upgrading/current-edition/)
+
+<!--alex ignore nuts-->
+Ember Octane represents a big shift in Ember's syntax, features, and mental models. There is, however, no need for a stop-the-world migration, Octane provides an opt-in method of migration where you can apply each core feature enhancement incrementally.
+
+If you are migrating an existing Ember application to Octane we have an excellent list of resources to help you out:
+
+- [Ember Guides: Octane Update Guide](https://guides.emberjs.com/release/upgrading/current-edition/)
+- [The Ember Octane vs. Classic Ember Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)
+- [Ember Atlas: Recommended Migration Order](https://www.notion.so/Atlas-Recommended-Migration-Order-a22f948a7cce4e01896d674f727bee74)
+
+Whatever you end up doing, you won’t have to do it alone as members of the community have generously contributed [numerous codemods](https://github.com/ember-codemods) to help us to migrate efficiently.
+
+- [Angle Brackets Codemod](https://github.com/ember-codemods/ember-angle-brackets-codemod)
+- [Tracked Properties Codemod](https://github.com/ember-codemods/ember-tracked-properties-codemod)
+- [Native Class Codemod](https://github.com/ember-codemods/ember-native-class-codemod)
+- [No Implicit This or {{foo}} to {{this.foo}} Codemod](https://github.com/ember-codemods/ember-no-implicit-this-codemod)
+- [ES5 Getter Codemod](https://github.com/ember-codemods/es5-getter-ember-codemod)
+- [Template Colocation Migrator](https://github.com/ember-codemods/ember-component-template-colocation-migrator)
+
+If you need help along the way, visit the [Ember Community chat and forums](https://emberjs.com/community/).
+
+---
+
+## [Community feedback from Outdoorsy 👏](https://www.outdoorsy.com)
+
+Wondering about who has been using Ember Octane features in production?
+
+We’ll spend some time over the next several issues focusing on projects and teams who are using Ember Octane in their apps.
+
+Today’s focus is on [**Outdoorsy**](https://www.outdoorsy.com), a peer-to-peer marketplace that connects RV and camper van owners with campers encouraging them to get into the great outdoors. Outdoorsy maintains three large-scale Ember apps for their flagship web product.
+
+The developers at Outdoorsy have been using Octane features in production since September and have reportedly had a super enjoyable experience. [Jaco Joubert @jacojoubert](https://github.com/jacojoubert), a Front-End Engineer at Outdoorsy, says "Octane has allowed us to build highly interactive experiences that are well tested in an unbelievably short time. Leveraging Octane we rebuilt our entire [search experience](https://www.outdoorsy.com/rv-search) from the ground up in less than five weeks, something which would have taken orders of magnitude longer before."
```

Comment:
Added parentheses around GitHub handler for Jaco.
```suggestion
The developers at Outdoorsy have been using Octane features in production since September and have reportedly had a super enjoyable experience. [Jaco Joubert (@jacojoubert)](https://github.com/jacojoubert), a Front-End Engineer at Outdoorsy, says "Octane has allowed us to build highly interactive experiences that are well tested in an unbelievably short time. Leveraging Octane we rebuilt our entire [search experience](https://www.outdoorsy.com/rv-search) from the ground up in less than five weeks, something which would have taken orders of magnitude longer before."
```